### DISCUSSION TITLE
Station Data

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#5574] [Q&A]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #5d9355d (2022-07-05 8:37) 

### What happened?

I can create , mount more than one volume ?

in this case I have an initial hd of 240 gb and I acquired another volume of 200 gb

Here comes my doubt, I configured with

services:
   web:
     volumes:
       - /path/on/host/computer:/var/azuracast/stations

But it only takes a path, in which I define above, I can in this same file apply two different volumes, HD1 and HD2 and in the creation of the account point to which HD I want to use?

### Relevant log output

```shell
****
```

**Comment by paulcjkp** [comment]
I'm not sure if you can use both at the same time, but information on mounting another volume are available in the docs, under Docker.  https://docs.azuracast.com/en/administration/docker

**Reply by Marcioghiraldelli** [reply]
I'm not sure if you can use both at the same time, but information on mounting another volume are available in the docs, under Docker. https://docs.azuracast.com/en/administration/docker

—
